routing/dht/dht_test.go

```diff
@@ -136,10 +136,10 @@ func TestValueGetSet(t *testing.T) {
 
 	connect(t, ctx, dhtA, dhtB)
 
-	ctxT, _ := context.WithTimeout(ctx, time.Second)
-	dhtA.PutValue(ctxT, "/v/hello", []byte("world"))
```

Comment:
why remove this timeout? we expect this to happen immediately, if it hangs, then thats wrong and we want to know about it.